core/shared/src/main/scala/fs2/Chunk.scala

```diff
@@ -343,7 +348,6 @@ object Chunk {
     def apply(i: Int) = values(offset + i)
     protected def splitAtChunk_(n: Int): (Chunk[O], Chunk[O]) =
       Boxed(values, offset, n) -> Boxed(values, offset + n, length - n)
-    override def map[O2](f: O => O2): Chunk[O2] = seq(values.map(f))
```

Comment:
It surprises me that we don't have specialized implementations of map for array backed chunks. Maybe that's a good optimization for a separate PR.